app/views/requests.erb

```diff
@@ -1,49 +1,56 @@
 <h1>Requests</h1>
 
-<h2>Requests I've made</h2>
-<ul>
-  <% if @guest_requests.empty? %>
-    <li>It seems like you haven't made any request yet</li>
-  <% else %>
-    <% @guest_requests.each do |guest_request| %>
-      <% if guest_request.booking_start && guest_request.booking_end %>
-        <li>Property name:  <%= Space.get(guest_request.space_id).name %> </li>
-        <li>Host info:  <%= User.get(guest_request.host_id).name %> </li>
-        <li>Status:  <%= guest_request.status.capitalize %> </li>
-        <li>From  <%= guest_request.booking_start %> to <%= guest_request.booking_end %> </li>
+<div class="flexbox">
+  <section class="boxitem">
+    <h2>Requests I've made</h2>
+      <% if @guest_requests.empty? %>
+        <ul>
+          <li>It seems like you haven't made any request yet</li>
+        </ul>
+      <% else %>
+        <% @guest_requests.each do |guest_request| %>
```

Comment:
Just realised we will need to change the logic below to conform with the original master since we now include booking ranges rather than single booking dates. It should look like this instead:

```
<% @guest_requests.each do |guest_request| %>
  <% if guest_request.booking_start && guest_request.booking_end %>
    <li>Property name:  <%= Space.get(guest_request.space_id).name %> </li>
    <li>Host info:  <%= User.get(guest_request.host_id).name %> </li>
    <li>Status:  <%= guest_request.status.capitalize %> </li>
    <li>From  <%= guest_request.booking_start %> to <%= guest_request.booking_end %> </li>
  <% end %>
<% end %>
```